src/core/import-maps.js

```diff
@@ -20,3 +21,7 @@ export const marked = _marked;
 /** @type {import("pluralize")} */
 // @ts-ignore
 export const pluralize = _pluralize;
+
+/** @type {import("sniffy-mimetype")} */
+// @ts-ignore
```

Comment:
Do we need this ts-ignore?

Reply:
dunno, will try without it. Was cargo cutting based on the others.

Reply:
Yeah, it gets upset. 

![Uploading Screen Shot 2021-07-22 at 7.21.59 pm.png…]()

Reply:
I wonder if this is because the dependency doesn't bundle a typescript file? Or because it's lacking the types. 

@saschanaz, I see that the WebIDL2.js import doesn't need the `tsignore`... what magic are you doing there? 🪄

Reply:
dunno, maybe because it has no `/** @type {import()} */`?